src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+/**
+ * Represents the User Interface in DukeProject.
+ */
+public class Ui {
+    private String logo;
+    private String lineBreak;
+
+    /**
+     * Prints out the introductory message for DukeProject.
+     */
+    public void getIntroduction() {
+        System.out.println(lineBreak);
+        System.out.println("Hello I am \n" + logo
+                + "\nWhat can I do for you?");
+        System.out.println(lineBreak);
+    }
+
+    /**
+     * Prints out the exit message for DukeProject.
+     */
+    public void exit() {
+        System.out.println(lineBreak);
+        System.out.println("Bye. Hope to see you again soon!");
```

Comment:
Since System.out.println is used often, perhaps it may save you time to define print() method which does the same thing. This could help in the amount of words needed to be typed.